packages/grafana-data/src/themes/zIndex.ts

```diff
@@ -1,7 +1,6 @@
 // We need to centralize the zIndex definitions as they work
 // like global values in the browser.
 export const zIndex = {
-  activePanel: 999,
```

Comment:
think we have to leave this in for a bit, will break scene apps if we just remove it in one go as it's used by scenes lib.

Reply:
should we wait until this this gets updated in scenes and the dep is bumped?

Reply:
@leeoniya it wont be enough, as all scene app plugins would also have to be updates with latest scenes version so it will take time, better leave this here for now

Reply:
:+1: